uw-frame-components/portal/search/partials/search.html

```diff
@@ -7,7 +7,7 @@
             aria-label="Search bar enter the app you are looking for"
             focus-me="showSearchFocus"
             ng-show="$storage.typeaheadSearch"
-            typeahead="title as match.title for match in filterMatches | limitTo:10"
+            uib-typeahead="title as match.title for match in filterMatches | limitTo:10"
```

Comment:
@timlevett shouldn't the rest of these change too? (lines 11+)

We're using it in a totally similar way to [search for projects](https://git.doit.wisc.edu/adi-ia/star/blob/master/star-app/src/main/webapp/my-app/timecard/timecard.html#L120)